docs/reference/monitoring/configuring-elastic-agent.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[configuring-elastic-agent]]
+== Collecting {es} monitoring data with {agent}
+
+[subs="attributes"]
+++++
+<titleabbrev>Collecting monitoring data with {agent}</titleabbrev>
+++++
+
+In 8.5 and later, you can use {agent} to collect data about {es} and ship it to
+the monitoring cluster, rather than <<configuring-metricbeat,using {metricbeat}>>
+or routing it through exporters as described in <<collecting-monitoring-data>>.
+
+[discrete]
+=== Prerequisites
+
+* (Optional) Create a monitoring cluster as described in <<monitoring-production>>.
+* If you decide to use a monitoring cluster, add an {es} output under
```

Comment:
Note to self: Remove this item to match the changes made in https://github.com/elastic/kibana/pull/152634